storages/memory-storage/tests/memory_storage.rs

```diff
@@ -94,3 +100,101 @@ fn memory_storage_transaction() {
     test!(glue "COMMIT", Ok(vec![Payload::Commit]));
     test!(glue "ROLLBACK", Ok(vec![Payload::Rollback]));
 }
+
+#[test]
+fn memory_storage_function() {
```

Comment:
This test doesn't need to be only for `memory_storage`.
Could we move this to `test-suite`? Then all the storage makers can use this test.